doc/plugins/proxy-rewrite.md

```diff
@@ -35,7 +35,7 @@ The `proxy-rewrite` is an upstream proxy information rewriting plugin, which sup
 
 | Name      | Type          | Requirement | Default | Valid             | Description                                                  |
 | --------- | ------------- | ----------- | ------- | ----------------- | ------------------------------------------------------------ |
-| scheme    | string        | optional    | "http"  | ["http", "https"] | Upstream new `schema` forwarding protocol.                   |
+| scheme    | string        | optional    | "http"  | ["http", "https"] | Deprecated. It's recommended to set the proxy `scheme` in the Upstream object's `scheme` field instead.|
```

Comment:
I think in the `Description` part, a description should be added or the original description should be retained.
such as:  
Upstream new `schema` forwarding protocol (Deprecated. It's recommended to set the proxy `scheme` in the Upstream object's `scheme` field instead).

Reply:
@Firstsawyou 
Updated.